OpenEdXMobile/src/main/java/org/edx/mobile/inapppurchases/BillingProcessor.kt

```diff
@@ -209,6 +209,7 @@ class BillingProcessor @Inject constructor(@ApplicationContext val context: Cont
         private val TAG = BillingProcessor::class.java.simpleName
         private const val RECONNECT_TIMER_START_MILLISECONDS = 1L * 1000L
         private const val RECONNECT_MAX_COUNT = 3 // retry connection max count
+        const val MICROS_TO_UNIT = 1_000_000 // 1,000,000 micro-units equal one unit of the currency.
```

Comment:
No need for a full-stop. It will prevent the editor from the comment to the new line as well.